src/resources.cpp

```diff
@@ -102,6 +108,31 @@ std::vector<string> DirectoryResourceProvider::findResources(string searchPatter
 
 void DirectoryResourceProvider::findResources(std::vector<string>& found_files, const string path, const string searchPattern)
 {
+#ifdef _WIN32
+    WIN32_FIND_DATAA data;
+    string search_root(basepath + path);
+    assert(search_root.endswith("/"));
```

Comment:
This changes the API to have a required slash at the end, which I do not want to enforce. (So this needs to be fixed before this pull request is ready)

It can also never be an assert, as in some cases (--mod) this is a user supplied name)